pkg/monitortests/clusterversionoperator/legacycvomonitortests/operators.go

```diff
@@ -52,6 +52,12 @@ func testStableSystemOperatorStateTransitions(events monitorapi.Intervals, clien
 		isSingleNode = false
 	}
 
+	isTwoNodeFencing, err := isTwoNodeFencingCheck(clientConfig)
+	if err != nil {
+		logrus.Warnf("Error checking for TwoNodeFencing Node configuration on upgrade (unable to make exception): %v", err)
```

Comment:
I would probably rephrase this:
"Error checking for DualReplica controlPlaneTopology"